src/mscorlib/src/System/Runtime/CompilerServices/TaskAwaiter.cs

```diff
@@ -129,7 +135,8 @@ internal static void ValidateEnd(Task task)
         /// the await on the task, and throws an exception if the task did not complete successfully.
```

Comment:
If this method returns rather than throws, the comment is wrong.

Reply:
Fixed